docs/contributing/style-guide.md

```diff
@@ -1,7 +1,595 @@
+
+<table class="code">
+  <tbody>
+    <tr>
+      <td>
+
+**Markdown**
+
+</td>
+      <td>
+
+    ```java reference title="Example Test"
+    https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
+    ```
+
+</td>
+</tr>
+<tr>
+<td>
+
+**Display**
+
+</td>
+      <td>
+
+```java reference title="Example Test"
+https://github.com/saucelabs-training/demo-java/blob/website-1.0/selenium-examples/src/test/java/com/saucedemo/junit5/SauceBindingsTest.java#L35-L43
+```
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Admonitions
+
+There are 4 types of [Docusaurus Admonitions](https://docusaurus.io/docs/markdown-features/admonitions):
+* Note - Relevant information.
+* Tip - A user should do this.
+* Caution - A user should pay attention to this.
+* Warning - A user might do something dangerous!
+
+<table class="code">
+  <tbody>
+    <tr>
+      <td>
+
+**Markdown**
+
+</td>
+      <td>
+
+```markdown
+:::note
+Relevant information for you.
+:::
+
+:::tip
+You should do this.
+:::
+
+:::caution
+You should probably pay attention to this.
+:::
+
+:::warning
+You are about to do something dangerous!
+:::
+```
+
+</td>
+</tr>
+<tr>
+<td>
+
+**Display**
+
+</td>
+      <td>
+
+:::note
+Relevant information for you.
+:::
+
+:::tip
+You should do this.
+:::
+
+:::caution
+You should probably pay attention to this.
+:::
+
+:::warning
+You are about to do something dangerous!
+:::
+
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Badges
+
+Badges are color-coded images that alert a reader to special considerations for a given section. 
+We currently support 2 types of badges:
+
+GOLD indicates that a page is deprecated or out-of-date in some way, for example:
+
+* DEPRECATED
+
+BLUE indicates that the feature being documented is limited in its scope or availability, for example:
+
+* ENTERPRISE ONLY
+* IOS ONLY
+* LIVE TESTING ONLY
+* BETA
+* EARLY ACCESS
+
+<table class="code">
+  <tbody>
+    <tr>
+      <td>
+
+**HTML**
+
+</td>
+      <td>
+
+```html
+<p><span className="sauceDBlue">BETA</span></p>
+<p><span className="sauceDBlue">ENTERPRISE ONLY</span></p>
+<p><span className="sauceDBlue">IOS ONLY</span></p>
+<p><span className="sauceDBlue">LIVE TESTING ONLY</span></p>
+<p><span className="sauceDBlue">EARLY ACCESS</span></p>
+
+<p><span className="sauceGold">DEPRECATED</span></p>
+```
+
+</td>
+</tr>
+<tr>
+<td>
+
+**Display**
+
+</td>
+      <td>
+<p><span className="sauceDBlue">BETA</span></p>
+<p><span className="sauceDBlue">ENTERPRISE ONLY</span></p>
+<p><span className="sauceDBlue">IOS ONLY</span></p>
+<p><span className="sauceDBlue">LIVE TESTING ONLY</span></p>
+<p><span className="sauceDBlue">EARLY ACCESS</span></p>
+
+<p><span className="sauceGold">DEPRECATED</span></p>
+</td>
+    </tr>
+  </tbody>
+</table>
+
+## Cards
+
+For overview pages that have 4 categories, we often use these Boxes. Note that you can't use Markdown inside this html.
```

Comment:
Do we want to show the boxes that we have defined for a top or bottom row that is full width, or is that overkill? See this page for an example: https://docs.saucelabs.com/performance

Reply:
tbh, I almost didn't include this section because I doubt any potential external contributors will use these, and if Saucers want to use it, they'll just copy from another page that. :)
So, this is totally your call on whether it belongs as an example in the style guide.